obspy/clients/fdsn/tests/test_client.py

```diff
@@ -335,19 +335,46 @@ def test_iris_event_catalog_availability(self):
         Tests the parsing of the available event catalogs.
         """
         self.assertEqual(set(self.client.services["available_event_catalogs"]),
-                         set(("ANF", "GCMT", "TEST", "ISC", "UofW",
-                              "NEIC PDE")))
+                         set(("GCMT", "ISC", "NEIC PDE")))
 
     def test_iris_event_contributors_availability(self):
         """
         Tests the parsing of the available event contributors.
         """
+        # contributors to IRIS as of 2016-12-07
+        results = {
+            'us_ath', 'us_sja', 'ldg', 'rom', 'oh', 'hdc', 'EPT', 'ak,us',
+            'TIR', 'HEL', 'BGS', 'us_pgc', 'AGS', 'pgc', 'PRE', 'us_buc', 'ld',
+            'TAP', 'ott', 'NNC', 'wel', 'wes', 'MRB', 'LIM', 'sko', 'spec',
+            'us_ott', 'pivs', 'USBR', 'D', 'H', 'pr,us', 'L', 'BOU', 'WES',
+            'P', 'us_mdd', 'zamg', 'bji', 'THE', 'GL', 'GII', 'guc', 'AUST',
+            'rsnc', 'HVO', 'trn', 'NIC', 'DBN', 'GEN', 'ismpkansas', 'us_lim',
+            'RYD', 'MEL', 'us_rom', 'us_isk', 'GCMT', 'us', 'GUC', 'A', 'ismp',
+            'us_snet', 'BEO', 'FDF', 'LAK', 'HV', 'mdd', 'us_bgs', 'us_car',
+            'HDC', 'aust', 'SSNC', 'BER', 'SKO', 'us_ecx', 'bdf', 'LDG', 'ROM',
+            'FBB', 'PAL', 'MAC', 'BUT', 'HON', 'nic', 'BUC', 'JMA', 'SCP',
+            'TRI', 'pr', 'C', 'BDF', 'G', 'TRN', 'HOJ', 'BYKL', 'CSC', 'UPA',
+            'GRF', 'ber', 'S', 'tul', 'O', 'AEIC', 'qui', 'DDA', 'LJU',
+            'us_inmg', 'CSS', 'ecx', 'zur', 'CASC', 'TUL', 'MB', 'UU', 'PGC',
+            'UW', 'ath', 'SPEC', 'us_teh', 'uu', 'bgs', 'MPR', 'iscgemsup',
+            'pdg', 'ZAG', 'ALG', 'sja', 'unm', 'ZUR', 'teh', 'ci', 'us_rsnc',
+            'ucr', 'sof', 'zag', 'ATH', 'DOE', 'ISK', 'REN', 'INMG', 'DMN',
+            'ISC', 'us_wel', 'IRSA', 'TEH', 'TEI', 'NM', 'NN', 'BLA', 'JSN',
+            'NC', 'us_unm', 'MDD', 'snet', 'W', 'UPP', 'SEA', 'BSE', 'UCR',
+            'TVA', 'gii', 'ECX', 'NDI', 'CI', 'B', 'UNM', 'sors', 'SJR', 'CLE',
+            'J', 'OGSO', 'DNK', 'us_qui', 'R', 'inmg', 'V', 'us_ldg', 'buc',
+            'SJA', 'CNRM', 'WAR', 'PDA', 'isk', 'ZAMG', 'us_hlw', 'casc', 'mb',
+            'CSEM', 'car', 'uw', 'INDR', 'BJI', 'SPE', 'SNET', 'HLW', 'bgrd',
+            'BRK', 'QUI', 'EXP', 'NSSP', 'SE', 'PAS', 'OBN', 'tap', 'CAR',
+            'ak', 'official', 'AEI', 'SNM', 'US', 'CERI', 'iscgem', 'us_thr',
+            'us_gcg', 'RSNC', 'nm', 'nn', 'SOF', 'csem', 'lim', 'nc', 'tir',
+            'PDG', 'us_the', 'PPT', 'GOM', 'E', 'MACR', 'AG', 'TEIC', 'I',
+            'hv', 'M', 'LD', 'THR', 'SLM', 'rey', 'U', 'DHMR', 'STR', 'us_tul',
+            'WEL', 'SLC', 'us_guc', 'DJA', 'BHL', 'us_aust', 'RSPR', 'thr',
+            'T', 'GFU', 'SDD', 'dja', 'the', 'us_trn', 'OTT', 'se', 'ARO'}
```

Comment:
This will be super fragile and I suspect this will keep changing. How about to just test if the `self.client.services["..."]` contains anything at all? And then maybe augment it with a mock test that does the actual parsing?